camel-cdi-integration/src/main/java/io/silverware/microservices/providers/camel/CamelCdiContextFactory.java

```diff
@@ -62,8 +61,7 @@ public CamelContext createCamelContext(final Context context) {
          if (cdiSilverService.isDeployed()) {
             log.info("CDI SilverService connected successfully!");
 
-            CamelContextMap camelContextMap = cdiSilverService.findByType(CamelContextMap.class);
-            return camelContextMap.getCamelContext("camel-1");
+            return cdiSilverService.findByType(CamelContext.class);
```

Comment:
Can you please explain this change. How is `cdiSilverService.findByType(CamelContext.class)`equivalent to `camelContextMap.getCamelContext("camel-1")`, where does the "camel-1" context name come from?